Files/ar_accept_or010204.csv

```diff
@@ -1,5 +1,5 @@
-name01#0,"""アスタルテ"""
-text01#0,"""押印完了しました。<br>お疲れ様でした。"""
-11_voice_shop_414#0,"""次の課題は、惑星ナベリウス<br>森林エリアのフリーフィールドです。"""
-text03#0,"""<c bb0000>「森林探索」<c>の<br>クリアが目的となります。"""
-text04#0,"""それでは、頑張って下さい。"""
+name01#0,"""Astarte"""
+text01#0,"""The seal has been completed.<br>Thank you for your hard work."""
+11_voice_shop_414#0,"""The next mission is on Planet Navelius<br>It is a free field of the new area"""
+text03#0,"""<c bb0000>Mission Objective:<c>Clear<br>「Forest Exploration」"""
```

Comment:
```suggestion
text03#0,"""<c bb0000>Mission Objective :<c> Clear<br>\""Forest Exploration\"""""
```
Take note on the spacing and line break (<br>), also the 「」 character should be replaced by a quote (") in string format (\"").